css/qmwp-search-correlations.css

```diff
@@ -0,0 +1,286 @@
+.fieldsContainer {
+    font-size:0;
+    max-width: 100%;
+    position: relative;
+}
+
+.searchsubmit{
+    padding:6px 20px !important;	
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/964746528/issues/source?bid=2836188&fileBranchId=2859501#l8)